util/Seccomp.c

```diff
@@ -59,6 +60,10 @@ static void catchViolation(int sig, siginfo_t* si, void* threadContext)
 {
     printf("Attempted banned syscall number [%d] see doc/Seccomp.md for more information\n",
            GET_SYSCALL_NUM(si));
+#ifndef si_syscall
```

Comment:
I think it's `siginfo_t.si_syscall` or something like that. siginfo_t is a big union with all kinds of stuff.